requirements.txt

```diff
@@ -53,3 +53,7 @@ editdistance
 
 # Tutorial notebooks
 jupyter
+
+# For pretrained model weights
```

Comment:
Can you add this requirement to `setup.py` also?

Reply:
ok